packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -11,19 +11,29 @@ interface ComboboxContentProps {
   readonly options: string[];
 
   /**
-   * onSelection handler.
+   * onSelect handler.
    */
-  readonly onSelection: (selection: string) => void;
+  readonly onSelect: (selection: string) => void;
 
   /**
    * actions
    */
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * pre selected option
+   * @default ""
+   * @optional
+   * @type string
+   * */
+  readonly selected?: string;
 }
 
 export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
```

Comment:
I actually don't mind the `prop` vs destructured (`{ onChange }`) as much but since we've been very consistent around it too. Does this land on "depends on the context" style or "nah, we need to be consistent" side? I actually don't know 🤔

Reply:
I strongly prefer the `prop.thing` syntax and avoid destructuring (when it comes to props specifically, hooks and other stuff is fine)

it's definitely from my old Ember days where you'd be in a component and see `option` but have absolutely no way of knowing where `option` came from

is it 
- an internal property?
- a property being passed in via Ember's version of props
- a mixin?

new Ember solves this by prefixing props with `@thing` and then internal state with `this.thing` making it much quicker to know where the values are coming from, which I think is really nice!